### ISSUE TITLE
Joiner Penalty does not persist between servers, making game shopping trivial 

### ISSUE THREAD

**Issue by RonSpawnson** [tagpro-game/tagpro-issues#274]
As evidenced by recent special events, the joiner penalty for dc'ing is easily circumvented by switching servers, leading to unpenalized game shopping.

**Comment by Nick-Riggs** [comment]
Yeah, I might try to stop this next time around. The thing is, there will still be crafty ways around it.
